cli/src/lib.rs

```diff
@@ -271,25 +271,21 @@ impl Iroha {
 
         let kura_thread_handler = Kura::start(Arc::clone(&kura));
 
-        let sumeragi = Arc::new(
-            // TODO: No function needs 10 parameters. It should accept one struct.
-            Sumeragi::new(
-                &config.sumeragi,
-                events_sender.clone(),
-                wsv,
-                transaction_validator,
-                Arc::clone(&queue),
-                Arc::clone(&kura),
-                network.clone(),
-            ),
+        // TODO: No function needs 10 parameters. It should accept one struct.
```

Comment:
I don't insist on fixing this right now, but considering the changes are already made, it could be useful.